Source/FakeItEasy/Core/DefaultFakeWrapperConfigurer.cs

```diff
@@ -6,21 +6,27 @@ namespace FakeItEasy.Core
     /// Handles configuring of fake objects to delegate all their calls to a wrapped instance.
     /// </summary>
     internal class DefaultFakeWrapperConfigurer
-        : IFakeWrapperConfigurer
     {
+        public DefaultFakeWrapperConfigurer(object wrappedObject)
+        {
+            this.WrappedObject = wrappedObject;
+        }
+
+        public object WrappedObject { get; private set; }
+
+        public ISelfInitializingFakeRecorder Recorder { get; set; }

```

Comment:
I think the mutable field or some variant of it is the way to go. We can't know that there's a recorder when we create the wrapper, it seems we need to add one later (as we are now), replace the wrapper with one that has a recorder, which means we're mutating the Wrapper field in FakeOptions, or (as was done before) have a parallel mutable field in FakeOptions. 
I think the current approach is better than the other two, or at least no worse.